### PR TITLE
feat: add single-video page

### PR DESCRIPTION
**Pull request by Bhakti199**
Here I added single video page.

Screenshot (1351)

### CODE REVIEW

File: src/Pages/SingleVideoPage/SingeVideoPage.jsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import { useParams } from "react-router-dom";
+import { useMainContext } from "../../Context/Index";
+import { Sidebar } from "../../Components/Index";
+export const SingeVideoPage = () => {
+  const videoId = useParams();
+  console.log(videoId);
+  const { videos } = useMainContext();
+  console.log(videos);
+  let video = videos.find((item) => item._id === videoId.videoId);
+  console.log(video);
```
Comment (tushartiwari7) [inline src/Pages/SingleVideoPage/SingeVideoPage.jsx]: remove console logs.

File: src/Pages/SingleVideoPage/SingeVideoPage.jsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    <>
+      <div className="single-video-page flex">
+        <div className="video-iframe flex">
+          <iframe
+            src={`https://www.youtube.com/embed/${videoId.videoId}`}
+            title="YouTube video player"
+            frameBorder="0"
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+            allowFullScreen
+          ></iframe>
```
Comment (tushartiwari7) [inline src/Pages/SingleVideoPage/SingeVideoPage.jsx:25]: This is good that you are getting just the videoId from BE

**Review by tushartiwari7 — APPROVED** [review]
Added some comments, have a look.

**Review by mariyasada — APPROVED** [review]
everything looks good to me just remove consoles

**Pull request merged**